assets/constants.ts

```diff
@@ -6811,6 +6811,22 @@ export default {
         }
       ]
     },
+    "FEEDBACK_UPDATES": {
+      "ROUTE": "feedback-updates",
+      "TITLE": "Feedback Updates",
+      "META": [
+        {
+          "PROPERTY_TYPE": "itemprop",
+          "PROPERTY_VALUE": "description",
+          "CONTENT": "With Oppia,learn and give feedback to improve the lessons"
```

Comment:
Add a space after the comma, ditto below.

Reply:
Done